src/util/createInstantSearchComponent.js

```diff
@@ -0,0 +1,60 @@
+import { createSuitMixin } from '../mixins/suit';
+import { version } from '../../package.json'; // rollup does pick only what needed from json
```

Comment:
We might not need this comment.

Reply:
It's more here so that someone later doesn't move it to a file for no reason